lib/cc/cli/runner.rb

```diff
@@ -18,7 +18,7 @@ def initialize(args)
       end
 
       def run
-        VersionChecker.new.check if check_version?
+        VersionChecker.new.check
```

Comment:
We still want this, to support `--no-check-version`. We're just going to flip the default to `--check-version` now that the cache is in place.